src/test/redux/actions/platform-actions.test.js

```diff
@@ -0,0 +1,270 @@
+      body: [{
+        id: '1',
+        name: 'Offering 1'
+      }]
+    }));
+
+    const expectedActions = [{
+      type: `${FETCH_PLATFORM_ITEMS}_PENDING`,
+      meta: { platformId: '1' }
+    }, {
+      meta: { platformId: '1' },
+      type: `${FETCH_PLATFORM_ITEMS}_FULFILLED`,
+      payload: [{ id: '1', name: 'Offering 1' }]
+    }];
+
+    return store.dispatch(fetchPlatformItems('1')).then(() => {
+      expect(store.getActions()).toEqual(expectedActions);
+    });
+  });
+
+  it('should dispatch correct actions after fetching platform items fails', () => {
+    const store = mockStore({
+      platformReducer: {
+        isPlatformDataLoading: false
+      }
+    });
+    apiClientMock.get('/r/insights/platform/topological-inventory/v0.0/sources/1/service_offerings', mockOnce({
+      status: 500
+    }));
+
+    const expectedActions = [
+      {
+        type: `${FETCH_PLATFORM_ITEMS}_PENDING`,
+        meta: { platformId: '1' }
+      }, {
+        type: `${ADD_NOTIFICATION}`,
+        payload: expect.objectContaining({ variant: 'danger' })
+      },
+      expect.objectContaining({ type: `${FETCH_PLATFORM_ITEMS}_REJECTED` })
+    ];
+
+    return store.dispatch(fetchPlatformItems('1')).catch(() => {
+      expect(store.getActions()).toEqual(expectedActions);
+    });
+  });
+
+  it('should dispatch correct actions after fetching multiple platforms items', () => {
+    const store = mockStore({
+      platformReducer: {
+        isPlatformDataLoading: false
+      }
+    });
+    apiClientMock.get('/r/insights/platform/topological-inventory/v0.0/sources/1/service_offerings', mockOnce({
+      body: [{
+        id: '1',
+        name: 'Offering 1'
+      }]
+    }));
+    apiClientMock.get('/r/insights/platform/topological-inventory/v0.0/sources/2/service_offerings', mockOnce({
+      body: [{
+        id: '2',
+        name: 'Offering 2'
+      }]
+    }));
+    apiClientMock.get('/r/insights/platform/topological-inventory/v0.0/sources/3/service_offerings', mockOnce({
+      body: [{
+        id: '3',
+        name: 'Offering 3'
+      }]
+    }));
+
+    const expectedActions = [{
+      type: `${FETCH_MULTIPLE_PLATFORM_ITEMS}_PENDING`
+    }, {
+      type: `${FETCH_MULTIPLE_PLATFORM_ITEMS}_FULFILLED`,
+      payload: {
+        1: [{ id: '1', name: 'Offering 1' }],
+        2: [{ id: '2', name: 'Offering 2' }],
+        3: [{ id: '3', name: 'Offering 3' }]
+      }
+    }];
+
+    return store.dispatch(fetchMultiplePlatformItems([ '1', '2', '3' ])).then(() => {
+      expect(store.getActions()).toEqual(expectedActions);
+    });
+  });
+
+  it('should dispatch correct actions after fetching multiple platforms items fails', () => {
+    const store = mockStore({
+      platformReducer: {
+        isPlatformDataLoading: false
+      }
+    });
+    apiClientMock.get('/r/insights/platform/topological-inventory/v0.0/sources/1/service_offerings', mockOnce({
```

Comment:
use TOPOLOGICAL_INVENTORY_API_BASE  here - and the rest of the places in this file where the base path is used

Reply:
Ah nice catch!